client/src/components/AdvancedSearch.js

```diff
@@ -136,9 +132,12 @@ const AdvancedSearch = ({
               </ButtonGroupContainerS>
             </FormGroup>
 
-            <FormControl defaultValue={text} className="hidden" />
+            {/* <FormControl defaultValue={text} className="hidden" /> */}
```

Comment:
It's useful (I find) to add `FIXME` comment in these cases, so they don't get lost (they show up in the tasks/todo list of my IDE).